Android/app/src/main/java/com/vigbag/android/MainActivity.kt

```diff
@@ -2,15 +2,15 @@ package com.vigbag.android
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import androidx.fragment.app.Fragment
 import com.vigbag.android.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding:ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setTheme(R.style.Theme_VigyBag)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
```

Comment:
no need for binding we can directly have `setContentView(R.layout.activity_main)`